requirements/base.txt

```diff
@@ -1,5 +1,5 @@
 attrs<19.0.0
-django==1.11.14
+django==1.11.15
```

Comment:
can we use `django>=1.11,<1.12` instead? I'm not clear why we would want to pin this to a minor version.

Reply:
My only reason to pin it was to [match edx-platform](https://github.com/edx/edx-platform/blob/master/requirements/edx/django.txt#L1), but yep, can change this.

Reply:
Blockstore uses python 3 and has its own virtual environment so I don't think we need to match.